cafee83/settings.py

```diff
@@ -81,12 +86,12 @@
 
 DATABASES = {
```

Comment:
DATABASES_URL

Reply:
https://stackoverflow.com/questions/51745898/set-the-database-url-environment-variable

![Uploading image.png…]()

Reply:
![image](https://github.com/mohitprajapat2001/Cafee83/assets/71551531/2bd0377e-b7a1-49d2-a34d-eaaaf99c5aff)